setup.bat

```diff
@@ -0,0 +1,162 @@
+
```

Comment:
where is the instruction to run this file?

Reply:
Will add the steps in the readme.md